Firestore/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Unreleased
 
 # 1.4.2
+- [fixed] Fixed an issue where query results were temporarily missing documents
+  that previously had not matched but had been updated to now match the query.
```

Comment:
It's OK to refer to issues in other repos, like we've done below. In this case it would be `https://github.com/firebase/firebase-android-sdk/issues/155`.

Reply:
Done.